assessment.rb

```diff
@@ -1,12 +1,92 @@
 require 'csv'
 # Assessment answers to:
 
+#question 1:
+rails g model License
+
 # question 2:
 
+#create add parameters to the migrate file create from above
+class CreateLicenses < ActiveRecord::Migration
+  def change
+    create_table :licenses do |t|
+      t.string :surname
+      t.string :given_name
+      t.string :gender
+      t.integer :height
+      t.integer :weight
+      t.date :dob
+      t.string :eye_color
+      t.string :hair_color
+
+      t.timestamps null: false
+    end
+  end
+end
+
+#in a new script (create_license.rb) add the following person:
+license = License.create(surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63, weight: 132, dob: '1943-09-26', eye_color: 'Brown', hair_color: 'Black')
+p license #displays that the above was created
+
+
+
 # question 3:
 
-# question 4:
+#create a new file /tasks/db.rake
+
+require 'csv'
+
+namespace :db do
+  namespace :populate do
+
+    ###Licenses###
+    desc 'Fill the licensees table with example data'
+    task licensees: :environment do
+      License.transaction do
+        CSV.foreach(Rails.root + 'data/licensees.csv',
+                    headers: true) do |license_row|
+          license = license_row.to_hash
+          next if License.exists? city #won't duplicate rows
+          License.create! License
+        end
+      end
+    end
+
+  end
+end
+
+# question 4: Use ActiveRecord to retrieve all the licensees with Hazel eye color and Brown or Black hair color.
+
+ License.find_by! eye_color: 'Hazel' && (hair_color == 'Brown' || hair_color == 'Black')
```

Comment:
I'm not sure if this will work. It looks like a mix of Hash condition syntax, e.g.
`.find_by! key1: value1, key2: value2`
and Array condition syntax, e.g.
`find_by! "key1 = ? AND key2 = ?", value1, value2`